book/caching.md

```diff
@@ -0,0 +1,33 @@
+Caching
```

Comment:
In the guide there is a relatively big article, covers a lot of topics at once. This recipe is a concrete solution for concrete task. Small and concise.
So include or not - depends on cookbook concept.